checkbox-ng/plainbox/impl/session/remote_assistant.py

```diff
@@ -719,7 +719,7 @@ def resume_by_id(self, session_id=None):
 
         result_dict = {
             "outcome": IJobResult.OUTCOME_PASS,
-            "comments": _("Automatically passed after resuming execution"),
+            "comments": "Automatically passed after resuming execution",
```

Comment:
I wish we could do this, but I feel like we should do all in one go

Reply:
Resurrected the call and mocked it in the UTs.